cql3/statements/modification_statement.hh

```diff
@@ -110,12 +110,21 @@ private:
         };
 
     uint64_t* _cql_modification_counter_ptr = nullptr;
+    /// Like `_cql_modification_counter_ptr' but incremented upon updates of "system" keyspace.
+    uint64_t* _cql_system_modification_counter_ptr = nullptr;
+
```

Comment:
Should have just one of these pointers, computed at prepare time.

Reply:
(or a different cql_stats structure passed by the caller, if we decide to make it depend on who initiated the query rather than the keyspace the query ended up affecting)